modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -1034,7 +1035,45 @@ sub _summarise_website_db {
                                                               };
       }
     }
-  }
+
+    ## Get biotype definitions
+    my $terms_endpoint = $ols.'/terms';
+    # NOTE: a term url passed as a parameter to the terms endpoint
+    # should use http, lack www, and be twice(!) url-encoded
+    my $biotype_url = 'http://ensembl.org/glossary/ENSGLOSSARY_0000025';
+    my $safe_biotype_url = uri_escape(uri_escape($biotype_url));
+    my $biotype_term_url = $terms_endpoint . '/' . $safe_biotype_url;
+ 
+    sub fetch_term {
+      my %result;
+      my $url = $_[0];
+      my $term = $self->_get_rest_data($url);
```

Comment:
When restarting the server, I am seeing the message: `Variable "$self" will not stay shared`. Not sure what that means; but most likely that's because I declare a subroutine within a subroutine. Don't know what the proper course of actions is (move the `fetch_term` and `fetch_children_terms` functions to the top level? pass `$self` to them as an argument?)

Reply:
I've never seen that error message before! Methods within an object should always be called on $self, ($self->fetch_term) and then you write "my $self = shift" as the first line of the method (or, for example, "my ($self, $arg) = @_;" if passing an argument).